Library/ViewModels/PledgeViewModel.swift

```diff
@@ -810,20 +810,45 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
 
     self.title = context.map { $0.title }
 
+    let trackCheckoutPageViewData = Signal
+      .zip(
+        project,
+        baseReward,
+        rewards,
+        selectedQuantities,
+        refTag,
+        initialAdditionalPledgeAmount,
+        pledgeTotal,
+        baseRewardShippingTotal,
+        context.filter { $0 == .pledge }
+      )
+
     // Tracking
 
-    initialDataUnpacked
-      .observeValues { project, reward, refTag, _ in
+    trackCheckoutPageViewData
+      .observeValues { project, baseReward, rewards, selectedQuantities, refTag, additionalPledgeAmount, pledgeTotal, shippingTotal, _ in
         let cookieRefTag = cookieRefTagFor(project: project) ?? refTag
```

Comment:
`guard context == .pledge else { return }` above this line.

Reply:
i think this should give the same desired behavior. test it out though.

Reply:
It will result in the same behaviour, I guess the issue here is `_` right?

Reply:
yeah this is mostly a personal opinion but I would prefer to know what is in the closure instead of omitting it.